tests/unit/test_control_plane.py

```diff
@@ -3,14 +3,14 @@
 from pinecone import Client
 import pytest
 from time import time
-
-from utils.utils import retry_assert
-from utils.remote_index import PodType
+import numpy as np 
+from ..utils.utils import retry_assert
+from ..utils.remote_index import PodType
 
 env = os.getenv('PINECONE_REGION')
 key = os.getenv('PINECONE_API_KEY')
 client = Client(key, env)
-INDEX_NAME_PREFIX = 'control-plane'
+INDEX_NAME_PREFIX = 'control-plane-' + str(np.random.randint(10000))
```

Comment:
The indexes are deleted, see `RemoteIndex.__exit__()`.

The reason for the random index number is to avoid to parallel tests (e.g. from two different PRs \ pushes) using the same index and interfering with each other. Especially _because_ each test deletes its on indexes upon exit.

Reply:
Got it, makes sense. Thanks for the context.